package.json

```diff
@@ -1,31 +1,25 @@
 {
-  "name": "kinto-elm-experiments",
+  "name": "elm-kinto",
   "version": "1.0.0",
-  "description": "Experimenting with Elm and Kinto.",
-  "main": "index.js",
+  "description": "A client to help making requests to a Kinto storage server",
   "scripts": {
-    "build": "node_modules/.bin/elm-make src/Main.elm --output=build/app.js",
-    "start": "node_modules/.bin/elm-reactor",
-    "debug": "node_modules/.bin/elm-live src/Main.elm --output=html/live/app.js --dir=html/live -- --debug",
-    "live": "node_modules/.bin/elm-live src/Main.elm --output=html/live/app.js --dir=html/live",
-    "publish-to-gh-pages": "npm run build && cp html/index.html build/ && node_modules/.bin/gh-pages --dist build/",
```

Comment:
so, it turns out that if you do `npm run build` for example in the root folder, it'll use the `elm-package.json` which is at the same level, instead of using the one in `examples/`, and thus fails.

That's why I created a project `package.json` file in the `examples/` folder, which has all the cool stuff, and a limited/simpler "library" one in the root folder.